web-client/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
-name = "nimiq-light-client"
+name = "nimiq-web-client"
```

Comment:
I think we should specify this crate only builds for wasm32

Reply:
[build]
target = "wasm32-unknown-unknown"

Reply:
Unfortunately, AFAIK this is not supported

Reply:
Tracking issue: https://github.com/rust-lang/cargo/issues/6179